database/seeders/SongSeeder.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Song;
+use Illuminate\Database\Seeder;
+
+class SongSeeder extends Seeder
+{
+  /**
+   * Run the database seeds.
+   */
+  public function run(): void
+  {
```

Comment:
Create a factory for songs. 
You can have an array in the factory for the spotify cover image url

Reply:
Created a separate factoy for the songs